shuttle-explorer/README.md

```diff
@@ -0,0 +1,34 @@
+# Shuttle Explorer
+
+Timeline visualiser for [Shuttle](https://github.com/awslabs/shuttle).
+
+## Building
+
+To use the extension, it is currently required to build it from source. Pre-requisites include:
```

Comment:
Do you know how much hassle it is to make an extension available in the VS Code store?